devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy

```diff
@@ -171,17 +172,28 @@ def refresh() {
 	def refreshCmds = zigbee.readAttribute(zigbee.TEMPERATURE_MEASUREMENT_CLUSTER, 0x0000) +
 			zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020)
 
-	return refreshCmds + zigbee.enrollResponse()
+	if(!isEcolink()) {
+		refreshCmds += zigbee.enrollResponse()
+	}
+
+	return refreshCmds
 }
 
 def configure() {
+	def configureCommands = []
 	// Device-Watch allows 2 check-in misses from device + ping (plus 1 min lag time)
 	// enrolls with default periodic reporting until newer 5 min interval is confirmed
 	sendEvent(name: "checkInterval", value: 2 * 60 * 60 + 1 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
-
+	if(isEcolink()) {
+		configureCommands += zigbee.iasZoneConfig(30, 60 * 30) + zigbee.batteryConfig(60 * 30, 60 * 30 + 1) + zigbee.temperatureConfig(30, 60 * 5) + refresh()
```

Comment:
I think we want to make the max temp reporting for temperature much higher than ias, so that periodic temp reports are less likely to bring us spontaneously back online. See https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/3487